setup.py

```diff
@@ -170,8 +170,7 @@ def get_files(directory, install_path, valid_suffixes):
             'psydata': ["Jinja2"],
             # TODO #2348: pytest currently fails with pylint >=3
             'test': ["pep8", "flake8", "pylint<3.0", "pytest-cov",
-                     "pytest-pep8", "pytest-pylint", "pytest-flakes",
-                     "pytest-xdist"],
+                     "pytest-pylint", "pytest-flakes", "pytest-xdist"],
```

Comment:
We don't use pep8, flake8, pytest-pylint, pytest-flakes I think (@arporter can correct me). The wiki and developer guide mentions pep8, but we actually use pycodestyle. Would it make sense to remove them all and maybe add pycodestyle and pytest-pycodestyle instead?

(althought I tried the second and was not working for me)

Reply:
Also remove the < 3.0 from pylint?

Reply:
We do use flake8?

Reply:
Sorry my grep missed the folder starting with .

@arporter So then we the rule is that we should use flake8 instead of pycodestyle?

Reply:
I saw the setup.cfg but didn't know what was it needed for.

"pylint" is still needed as test dependency since the wiki says the reviewer should use it.

Reply:
Okay, I'll put it back. I was just trying to see how much faster I could make the CI tests by skipping it. :P